### PR TITLE
[runtime:node] Use path library.

### PR DESCRIPTION
**Pull request by skiplabsdaniel**

### CODE REVIEW

File: prelude/ts/src/sk_node.ts [hunk 1]
```diff
@@ -20,7 +21,9 @@ class WrkImpl implements Wrk {
     url: URL,
     options: WorkerOptions | undefined,
   ): Wrk {
-    const filename = "." + url.pathname.substring(process.cwd().length);
+    const cwd = process.cwd();
+    const child = url.pathname.startsWith(cwd + path.sep);
+    const filename = child ? "." + url.pathname.substring(cwd.length) : url;
```
Comment (mbouaziz) [inline prelude/ts/src/sk_node.ts]: Another version:
```js
const filename = path.relative(process.cwd(), url.pathname);
```
I don't know which is better :shrug:

File: prelude/ts/src/sk_node.ts [hunk 2]
```diff
@@ -61,18 +64,21 @@ class Env implements Environment {
...
-      path = "." + url.pathname.substring(process.cwd().length);
+      const child = url.pathname.startsWith(cwd + path.sep);
+      filename = child ? "." + url.pathname.substring(cwd.length) : url.pathname;
     // @ts-ignore
     } else if (url && url.default) {
-      // @ts-ignore
-      path = "." + url.default.substring(process.cwd().length);
+      const aurl = url as any;
+      const child = aurl.default.startsWith(cwd + path.sep);
+      filename = child ? "." + aurl.default.substring(cwd.length): aurl.default;
```
Comment (mbouaziz) [inline prelude/ts/src/sk_node.ts]:
```suggestion
      const default = url.default as string;
      const child = default.startsWith(cwd + path.sep);
      filename = child ? "." + default.substring(cwd.length): default;
```
?

File: prelude/ts/src/sk_node.ts (hunk 1 above)
Comment (skiplabsdaniel) [reply prelude/ts/src/sk_node.ts]: I hesitated to do it, I asked myself the same question. It's true that it's more concise.

Comment (skiplabsdaniel) [reply prelude/ts/src/sk_node.ts]: But the if it's relative path it must start with "./"

**Review by mbouaziz — APPROVED** [review]

**Pull request merged**